openapi/src/parts/controlled_azure_database.yaml

```diff
@@ -0,0 +1,130 @@
+paths:
+  /api/workspaces/v1/{workspaceId}/resources/controlled/azure/databases:
+    parameters:
+      - $ref: '#/components/parameters/WorkspaceId'
+    post:
+      summary: Create a new controlled Azure Database
+      operationId: createAzureDatabase
+      tags: [ ControlledAzureResource ]
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/CreateControlledAzureDatabaseRequestBody'
+      responses:
+        '200':
+          $ref: '#/components/responses/CreateControlledAzureDatabaseResponse'
+        '403':
+          $ref: '#/components/responses/PermissionDenied'
+        '500':
+          $ref: '#/components/responses/ServerError'
+
+  /api/workspaces/v1/{workspaceId}/resources/controlled/azure/databases/create-result/{jobId}:
+    parameters:
+      - $ref: '#/components/parameters/WorkspaceId'
+      - $ref: '#/components/parameters/JobId'
+    get:
+      summary: Retrieve information about an Azure Database create job.
+      operationId: getCreateAzureDatabaseResult
+      tags: [ ControlledAzureResource ]
+      responses:
+        '200':
+          $ref: '#/components/responses/CreateControlledAzureDatabaseResponse'
+        '202':
+          $ref: '#/components/responses/CreateControlledAzureDatabaseResponse'
+        '403':
+          $ref: '#/components/responses/PermissionDenied'
+        '500':
+          $ref: '#/components/responses/ServerError'
+
+  /api/workspaces/v1/{workspaceId}/resources/controlled/azure/databases/{resourceId}:
+    parameters:
+      - $ref: '#/components/parameters/WorkspaceId'
+      - $ref: '#/components/parameters/ResourceId'
+    get:
+      summary: Get a controlled Azure Database resource
+      operationId: getAzureDatabase
+      tags: [ ControlledAzureResource ]
+      responses:
+        '200':
+          description: Response to get database
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/AzureDatabaseResource'
+        '403':
+          $ref: '#/components/responses/PermissionDenied'
+        '404':
+          $ref: '#/components/responses/NotFound'
+        '500':
+          $ref: '#/components/responses/ServerError'
+    delete:
+      summary: Delete a controlled Azure Database resource.
```

Comment:
This is a synchronous operation? How long does it take? 

We have generally regretted synchronous delete operations.

Reply:
this just deletes one thing, it is fast enough. Async APIs are a pain to consume so we should default to sync.